test/verify/check-login

```diff
@@ -48,6 +48,14 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
         m.start_cockpit()
         b.open("/system")
 
+        def assert_no_login():
+            users = m.execute("who | grep web | cat") # |cat for true return status
+            assert users == ""
```

Comment:
Please squash the commits, so that change and test stay together.

Please use unittest assertions instead of plain `assert`, and do the string handling on the Python side. Together that will give you much better failure output. Also, I think for this test case it makes sense to not just check for the type/terminal, but also for the user name, and pass that as an argument:

```py
out = m.execute("who")
self.assertNotIn("web console", out)
self.assertNotIn(user, out)
```

Finally, "login" is a bit confusing here, as this test checks  so many other aspects of the login. Maybe `assert_utmp()` or `assert_who()`?